src/App.tsx

```diff
@@ -1,14 +1,73 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
```
еhere is no need to specify simple types